docs/openapi.md

```
@@ -5461,8 +5468,17 @@ Serve the A2A Agent Card at the well-known location.
 This endpoint provides the agent card that describes Lightspeed's
 capabilities according to the A2A protocol specification.
 
-Returns:
-    AgentCard: The agent card describing this agent's capabilities.
+### Parameters:
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)